### PR TITLE
Prefab Fixes and NonPlayerHealth Polymorphism

### PR DESCRIPTION
**Pull request by Galaxy25**
#63

### CODE REVIEW

File: Assets/Scripts/RandomItemDrop.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ private void Start()
         potHealth = GetComponent<NonPlayerHealth>();
     }
 
-    private void Update()
+    public override void Death()
     {
         // Zero health makes it "break".
         if (potHealth.Health <= 0)
```
Comment (BrandonPacewic) [inline Assets/Scripts/RandomItemDrop.cs:22]: Do we still need to check for health being zero if NonPlayerHealth is handling this call?

**Comment by Galaxy25** [comment]
@BrandonPacewic Removed the if statement

**Review by BrandonPacewic — APPROVED** [review]
LGTM!

**Comment by BrandonPacewic** [comment]
I'm getting ready to merge and updating the change log - please @me if any other changes are pushed.

**Pull request merged**